FhirToDataLake/SynapseValidate/SynapseValidator.py

```diff
@@ -0,0 +1,199 @@
+import os
```

Comment:
The folder name could just be `synapse` instead of `SynapseValidate`, so we can put more related resources into it in future.